Chapters/1-8-Interlogue.md

```diff
@@ -0,0 +1,45 @@
+## Interlogue
+
+The magistrate re-entered the room while the cell guard held Kipnu's hands behind his back. In the light, Kipnu could see that the magistrate's eyes were wholly uninterested in his job. They were disappointed in Kipnu, but not surprised.
+
+"Kipnu Um Heiko. I find you guilty of assault on Fajro Um Holrag and interruption of his consciousness. The sentence shall be a period of two La'To in Ulukoma. Do you understand?"
+
+"Do I have a choice?"
+
+"Not really. Rante, an escort will be here just after first lunch to escort Kipnu to the docks."
+
+"What do I do with him 'til then?"
+
+"Do what you do best. Lock him up and take a nap."
+
+"Hah, you got it."
+
+He turned to Kipnu and said, "Kipnu. I hope that one day you might join us to prepare for the trials ahead. Until then, may Ulukoma quench whatever poison plagues your consciousness."
+
+"You think people actually listen to your self-righteous trash?"
+
+The magistrate simply sighed and walked out the door. Rante shoved Kipnu back into the dark room of cells and locked him in a cell next to Liffa.
+
+"Fajro, I hear the hero is going with you. When do you want me to call an escort?"
+
+"We won't be needing an escort."
+
+"You sure? Liffa broke into the senate. and I'm sure people might certainly try what Kipnu did."
+
+"I'm positive. The trouble is over."
+
+"Suit yourself. You're free to take hae whenever. Here's the key, and I've got armchains in the other room if you need them."
+
+"Thank you, sir. That will be all," said Fajro, accepting the key, and Rante rushed back to his desk.
+
+Kipnu sat still and silent while I shifted the corner of hae cell closest to him and stuck my hand out of the little window, tapping on the wall for him to do the same. He shifted in his cell and then I felt his hand touch mine. It was cold and at first I wasn't sure if it was him, but I could feel the textures and shapes were the same and it broke my heart.
+
+"You will come back," I said, "I know you will."
+
+"I will be waiting," Kipnu said, "Wherever you land, I'll be there."
+
+For several va'sa we held on like this, our tears falling into the stone floor where innumerable people before us had to face whatever shred of justice lived here in Shaki. Then, with a slow and deliberate movement, I let go and stood up. I turned toward Fajro, staring at him as if to ask if he was ready. Fajro stood up and without speaking he opened hae cell door. My feet led me out of the cell and next to Kipnu's arm, still hanging in the air out of the single porthole window of his cell. I knealt down and took his palm onto my head, felt the warmth trickle back into it.
+
+"I love you," I whispered.
+
+I stood back up, my fingers lingering on his, and looked to Fajro, who was getting ready to move. With his lantern in one hand and a cane in the other, Fajro gestured to hae and together, they stepped into the light, leaving Kipnu in the darkness.
```

Comment:
This is the first chapter so far that I didn't absolutely love. Trying to figure out why... (maybe it's completely unrelated... too many work meetings today). But as for the things within the text here are potential reasons:
- The POV switch to third person, was actually okay, since you labeled it as interlogue, I as a reader am prompted to expect something different.  However, the sudden switch back to Lif's first person is confusing. Since this chapter is so short, it would even be weirder if split is already small chapter into two. Hmm so what can we do:
  1. Keep the entire chapter in third person, find a way to show Lif's feelings by showing in action instead of telling by thoughts.
  2. Cut out the entire third person POV, and find a way to convey the important information in Kip's scene when they come back. For example:
  ```
  The magistrates come back with Kipnu, his hands bond as a guard shoves him into a cell adjacent to mine.
  "An escort will be here just after first lunch to escort Kipnu to the docks, bid your farewells now..."
  and etc...
  ```
  3. Since the previous chapter is super long and this one is super short. You can try to even them out, and just make this normal chapter. Then the brief POV switch will be in a larger chapter, and seem less jarring than within this small chapter.
 
Another bit that probably contributed to my dislike of this chapter has to do with mismatched expectations. Interludes are sections that stand out from the main flow of the text. That can be poetic verses, surreal passages, flashbacks, shifts in POV, etc. While you did the POV shift, I was expecting something more poetic, and that could have lead to the sour taste of this chapter. Though as I previously mentioned, I also had a lot of issues with the way the POV shift was executed.

Hope this is helpful.

Reply:
Good! Caus. rewrote it and it's an entirely different scene :D